Source/Fuse.Nodes/Visual.BeginRemove.uno

```diff
@@ -84,7 +84,7 @@ namespace Fuse
 			}
 			else
 			{
-				args.Remove();				
+				UpdateManager.AddDeferredAction(args.Remove);
```

Comment:
I still think this is the wrong time to change this timing. Swapping the order of two events is hard to reason about the consequences of, and we're already through testing.